contxt/services/iot.py

```diff
@@ -41,11 +41,45 @@ class IotService(ConfiguredApi):
             base_url="https://feeds.api.ndustrial.io/v1",
             client_id="iznTb30Sfp2Jpaf398I5DN6MyPuDCftA",
         ),
+        ApiEnvironment(
+            name="staging",
+            base_url="https://feeds-staging.api.ndustrial.io/v1",
+            client_id="m35AEcxD8hf65sq04ZU7yFxqpqVkKzES",
+        ),
     )
 
     def __init__(self, auth: Auth, env: str = "production", **kwargs) -> None:
         super().__init__(env=env, auth=auth, **kwargs)
 
+    def provision_field_for_feed(self, feed_id: int, field_obj: Field):
+        return self.post(f"feeds/{feed_id}/fields", data=field_obj.post())
```

Comment:
`field_obj` is redundant, of course it's an object. everything is
```suggestion
    def provision_field_for_feed(self, feed_id: int, field: Field):
        return self.post(f"feeds/{feed_id}/fields", data=field.post())
```